apps/pie-storybook/stories/pie-input.stories.ts

```diff
@@ -1,16 +1,23 @@
-import { html } from 'lit';
+import { html, nothing } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { action } from '@storybook/addon-actions';
 import { useArgs as UseArgs } from '@storybook/preview-api';
 
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-input';
-import { types, inputModes, InputProps } from '@justeattakeaway/pie-input';
+import { types, inputModes, InputProps as InputPropsOriginal } from '@justeattakeaway/pie-input';
 /* eslint-enable import/no-duplicates */
 
 import { type StoryMeta } from '../types';
 import { createStory, type TemplateFunction } from '../utilities';
 import '@justeattakeaway/pie-button';
+import '@justeattakeaway/pie-icons-webc/IconPlaceholder';
+
+// Extending the props type definition to include storybook specific properties for controls
+type InputProps = InputPropsOriginal & {
```

Comment:
nit: i think we have used the `Base` convention before in the codebase
so maybe we can have something like `InputBaseProps`